### PR TITLE
enhance:refine the range of chunk size config value(#29388)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #29388

### CODE REVIEW

File: pkg/util/paramtable/component_param.go [hunk 1]
```diff
@@ -1782,8 +1782,8 @@ func (p *queryNodeConfig) init(base *BaseTable) {
 		Version:      "2.0.0",
 		DefaultValue: "1024",
```
Comment (xiaofan-luan) [inline pkg/util/paramtable/component_param.go:1783]: why not change the default value to 128?

Comment (MrPresent-Han) [reply pkg/util/paramtable/component_param.go:1783]: I'm not sure about possible side effects to modify this default value

**Comment by MrPresent-Han** [comment]
rerun ut

**Pull request merged**